src/editor/editor.cpp

```diff
@@ -118,7 +118,11 @@ void Editor::update(float elapsed_time)
   }
 
   if (save_request) {
-    level->save(world->get_basedir() + "/" + levelfile);
+    if (world) {
+      level->save(world->get_basedir() + "/" + levelfile);
```

Comment:
I'll allow you to use the short if form here:
level->save(world ? (world->get_basedir() + "/" + levelfile) : levelfile);